### PR TITLE
Upgrade to pac4j v3 SNAPSHOT

### PR DESCRIPTION
**Pull request by mmoayyed**

### CODE REVIEW

File: src/main/java/org/pac4j/springframework/web/CallbackController.java [hunk 1]
```diff
@@ -50,7 +53,9 @@ public void callback(final HttpServletRequest request, final HttpServletResponse
         assertNotNull("config", config);
         final J2EContext context = new J2EContext(request, response, config.getSessionStore());
 
-        callbackLogic.perform(context, config, J2ENopHttpActionAdapter.INSTANCE, this.defaultUrl, this.multiProfile, this.renewSession);
+        callbackLogic.perform(context, config, J2ENopHttpActionAdapter.INSTANCE, this.defaultUrl,
+            this.saveInSession, this.multiProfile, this.renewSession,
+            config.getClients().getClients().get(0).getName());
```
Comment (mmoayyed) [inline src/main/java/org/pac4j/springframework/web/CallbackController.java]: Not sure about this part.

**Comment by mmoayyed** [comment]
Related https://github.com/pac4j/spring-webmvc-pac4j/pull/15

File: src/main/java/org/pac4j/springframework/web/CallbackController.java (hunk 1 above)
Comment (leleuj) [reply src/main/java/org/pac4j/springframework/web/CallbackController.java]: Nice try ;-) But in fact, this parameter is a default client (a specific property) you can define for this specific endpoint (especially requested by the JAX-RS implementation).
See: https://github.com/pac4j/j2e-pac4j/blob/master/src/main/java/org/pac4j/j2e/filter/CallbackFilter.java#L48

**Comment by mmoayyed** [comment]
Thanks for the review/help. All done.

**Comment by leleuj** [comment]
Excellent! Merging

**Pull request merged**